src/renderers/dom/shared/DOMProperty.js

```diff
@@ -197,33 +197,63 @@ var DOMProperty = {
   properties: {},
 
   /**
-   * Mapping from lowercase property names to the properly cased version, used
-   * to warn in the case of missing properties. Available only in __DEV__.
-   *
-   * autofocus is predefined, because adding it to the property whitelist
-   * causes unintended side effects.
-   *
-   * @type {Object}
-   */
-  getPossibleStandardName: __DEV__ ? {autofocus: 'autoFocus'} : null,
-
-  /**
-   * All of the isCustomAttribute() functions that have been injected.
+   * Some attributes are aliased for easier use within React. We don't
+   * allow direct use of these attributes. See DOMAttributeNames in
+   * HTMLPropertyConfig and SVGPropertyConfig.
    */
-  _isCustomAttributeFunctions: [],
+  aliases: {},
 
   /**
-   * Checks whether a property name is a custom attribute.
+   * Checks whether a property name is a writeable attribute.
    * @method
    */
-  isCustomAttribute: function(attributeName) {
-    for (var i = 0; i < DOMProperty._isCustomAttributeFunctions.length; i++) {
-      var isCustomAttributeFn = DOMProperty._isCustomAttributeFunctions[i];
-      if (isCustomAttributeFn(attributeName)) {
+  shouldSetAttribute: function(name, value) {
+    if (DOMProperty.isReservedProp(name)) {
+      return false;
+    }
+
+    if (value === null || DOMProperty.properties.hasOwnProperty(name)) {
```

Comment:
So this is the case where `shouldSetAttribute` seems like confusing naming, right? Since in case of `null` it won’t “set” the attribute but rather remove it. Is that right?

Do you think `shouldUseAttribute` would be better naming?